src/tasks/start-task.ts

```diff
@@ -41,12 +40,15 @@ export function setupStartTask(): void {
 
       logMain(`Starting Aragon app development...`)
 
-      const appEnsName = getAppEnsName(bre.network.name)
-      const appName = getAppName(bre.network.name)
-      const appId: string = getAppId(appEnsName)
+      const arapp = readArapp()
+      const appNameProduction = parseAppName(arapp)
+      const { shortName } = getAppNameParts(appNameProduction)
+      // Note: Since only the aragonpm.eth APM registry is deployed in development,
+      // all apps will be hosted there regardless of their name
+      const appName = `${shortName}.aragonpm.eth`
```

Comment:
LGTM 👍

Now that we have consistency. What you think in a new PR change `appName` for `appEnsName`?

Reply:
Since we cannot change arapp.json's `appName` property I would use this name as the standard for consistency